src/components/search/content-highlights/ContentHighlightSet.jsx

```diff
@@ -15,30 +15,34 @@ import HighlightedContentCard from './HighlightedContentCard';
 import { COURSE_RUN_AVAILABILITY } from '../../course/data/constants';
 import { useEnterpriseCustomer } from '../../app/data';
 
-const ContentHighlightSet = ({ highlightSet }) => {
-  const { data: enterpriseCustomer } = useEnterpriseCustomer();
-  const {
-    uuid: highlightSetUUID,
-    highlightedContent,
-    title,
-  } = highlightSet;
+function useHighlightedContent(highlightedContent) {
+  const archivedContent = useRef([]);
+  const activeContent = useRef([]);
 
-  const archivedContent = useMemo(() => [], []);
```

Comment:
[inform] This refactor primarily strives to move away from this `useMemo` implementation as its usage is a bit strange to create a memoized array and then push to it outside of the `useMemo`. Instead, it was abstracted into a `useHighlightedContent` hook and uses React refs instead.